scripts/build_fuzzers.py

```diff
@@ -0,0 +1,65 @@
+import subprocess
+import toml
+import typing
+import logging
+import tarfile
+import re
+import time
+
+REPO_DIR = '/home/runner/work/nearcore/nearcore/'
```

Comment:
nit: Sort
Let's be consistent and standarize the use of double quotes. Just apply `black` formatter that is the standard in the team.
https://pypi.org/project/black/

Reply:
nearcore uses yapf. Leaving as is until we standartise